internal/gps/result.go

```diff
@@ -10,6 +10,10 @@ import (
 	"path/filepath"
 )
 
+// NumDepTreeWorkers determines the number of
+// WriteDepTree worker goroutines
+var NumDepTreeWorkers = 2
```

Comment:
Let's add this as a parameter to `WriteDepTree()`. We've avoided package global state in gps so far, and want to continue that trend 😄